src/Tickets/Commerce/Admin_Tables/Orders_Table.php

```diff
@@ -361,6 +361,30 @@ public function column_status( $item ) {
 		ob_start();
 		?>
 		<mark class="tribe-tickets-commerce-order-status status-<?php echo esc_attr( $status->get_slug() ); ?>">
+			<?php
+			switch ( $status->get_slug() ) {
+				case 'completed':
+					?>
+					<span class="dashicons dashicons-yes"></span>
+					<?php
+					break;
+				case 'refunded':
+					?>
+					<span class="dashicons dashicons-undo"></span>
+					<?php
+					break;
+				case 'failed':
+					?>
+					<span class="dashicons dashicons-no-alt"></span>
+					<?php
+					break;
+				case 'pending':
+					?>
+					<span class="dashicons dashicons-clock"></span>
+					<?php
+					break;
```

Comment:
Let's simplify this with the variation only, outside the switch just use a single `<span class"dashicons %s"></span>, $variation.